packages/dso-toolkit/src/components/application-heading/application-heading.mixins.scss

```diff
@@ -4,11 +4,16 @@
 @use "application-heading.variables" as application-heading-variables;
 
 @mixin root() {
-  border-bottom: 1px solid colors.$grijs-20;
-  margin-bottom: units.$u2;
+  border-block-end: 1px solid colors.$grijs-20;
+  padding-block-end: units.$u2;
 
+  h1,
```

Comment:
Waar komt die `h1` vandaan?

Reply:
> Waar komt die `h1` vandaan?

Als ik me goed herinner, kwam ik op die plek niet alleen `h2`'s tegen, maar ook ergens een `h1`. in de voorbeeldpagina's dan.
als ik nou iets mis in de css-spec, dan is dat iets om alle headings mee te selecteren, zoals een `h` ;)

Reply:
haha, ja dat kan idd, maar is mij toch iets te expliciet.
Naja, we moeten ermee leren leven. Ik heb het er wel eens over gehad met Bert Bos, maar hij had geen plannen om hier wat aan te doen :)